docs/index.txt

```diff
@@ -76,6 +76,10 @@ JDBC Driver Class
 
 The Crate JDBC driver class is ``io.crate.client.jdbc.CrateDriver``.
 
+A connection is obtained from the DriverManager, e.g.::
```

Comment:
what about `Class.forName("io.crate.client.jdbc.CrateDriver");` ? isn't that also required?